runtime/compiler/control/rossa.h

```diff
@@ -77,6 +77,7 @@ typedef enum {
    compilationAOTNoSupportForAOTFailure            = 51,
    compilationAOTValidateTMFailure                 = 52,
    compilationILGenUnsupportedValueTypeOperationFailure = 53,
+   compilationAOTRelocationRecordGenerationFailure = 54,
```

Comment:
I believe there's a string array in rossa.cpp that you'll have to add to as well.

Reply:
updated the string array.